src/Plotly.NET.ImageExport/PuppeteerSharpRenderer.fs

```diff
@@ -12,6 +12,10 @@ open DynamicObj
 module PuppeteerSharpRendererOptions =
 
     let mutable launchOptions = LaunchOptions()
+    
+    // https://github.com/hardkoded/puppeteer-sharp/issues/1211
+    launchOptions.Args <- [| "--no-sandbox" |]
```

Comment:
Should it be linux-only?

Reply:
i do not think we need to hard-code this at all. I have used this lib on linux without setting this. as we expose the launch settings, this can be set by the user if needed.

Reply:
That's imo too obscure. I spent some time investigating it, and we leave it as it, not working for some linux users?

Reply:
is this on the contrary guaranteed to work everywhere else where it wasn't needed before? if so, i guess theres no harm

Reply:
Can't say for sure. Windows and Linux tests pass though. Is there some reason why we don't have those for MacOS btw?